template/.gitignore

```diff
@@ -11,3 +11,6 @@ test/unit/coverage
 test/e2e/reports
 selenium-debug.log
 {{/e2e}}
+{{#pug}}
+*.vue
```

Comment:
In our project we use pug file as template engine.
generate code should be ignore.

I never used pug-loader. I think I shoud try it first. 

this PR may not widely enough for community and looks a bad idea,
sorry for waste your time.